ChatQnA/benchmark_results.md

```diff
@@ -0,0 +1,161 @@
+# ChatQnA Benchmark Results
+
+## Overview
+
+ChatQnA deployed on a single node with ICX cores as the head node and supporting 8x Gaudi2 cards.
+This is based on OPEA v1.3 release helm charts and images using vLLM inferencing platform.
+
+## Methodology
+
+Tests scale concurrent users from 1 to 256, and each user send 4 queries. Measuring end to end (E2E) latency average for each query, time to first token (TTFT) average and time per output token (TPOT) average.
+
+## Hardware and Software Configuration
+
+| **Category**                      | **Details**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| --------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **System Summary**                | 1-node, 2x Intel(R) Xeon(R) Platinum 8380 CPU @ 2.30GHz, 40 cores, 270W TDP, HT On, Turbo On, NUMA 2, Integrated Accelerators Available [used]: DLB 0 [0], DSA 0 [0], IAA 0 [0], QAT 0 [0], Total Memory 1024GB (32x32GB DDR4 3200 MT/s [3200 MT/s]), BIOS ETM02, microcode 0xd0003b9, 8x Habana Labs Ltd., 4x MT28800 Family [ConnectX-5 Ex], 4x 7T INTEL SSDPF2KX076TZ, 2x 894.3G SAMSUNG MZ1L2960HCJR-00A07, Ubuntu 22.04.3 LTS, 5.15.0-92-generic. Software: WORKLOAD+VERSION, COMPILER, LIBRARIES, OTHER_SW. |
```

Comment:
"Software: WORKLOAD+VERSION, COMPILER, LIBRARIES, OTHER_SW"

Either actually specify the software, or drop this part.